gap/attr.gi

```diff
@@ -2675,3 +2675,27 @@ function(D)
   D := DigraphReflexiveTransitiveReduction(DigraphMutableCopyIfMutable(D));
   return DIGRAPHS_NonSemimodularPair(InNeighbours(D)) = fail;
 end);
+
+InstallMethod(JoinSemilatticeDigraphJoinTable, "for a digraph",
```

Comment:
I completely understand the rationale for the name, but I think `DigraphJoinTable` would better!

Reply:
Done